client/me/help/main.jsx

```diff
@@ -219,6 +224,51 @@ class Help extends React.PureComponent {
 		);
 	};
 
+	supportSessionCard = () => {
+		return (
+			<Card className="help__support-session-card">
+				<div className="help__support-session-text">
+					<h2 className="help__support-session-title">
+						{ this.props.translate( 'Schedule a support session' ) }
+					</h2>
+					<p className="help__support-session-description">
+						{ this.props.translate(
+							'Quick Start Support Sessions give you a way to talk to one of our Happiness Engineers via a screen share with audio.'
+						) }
+					</p>
+					<div className="help__support-session-actions">
+						<Button
+							className="help__support-session-action"
+							primary
+							href={ localizeUrl( 'https://wordpress.com/checkout/offer-quickstart-session/' ) }
+							onClick={ this.trackSupportSessionButtonClick() }
+						>
+							Schedule a session
+						</Button>
+						<Button
+							className="help__support-session-action is-link"
+							borderless
+							href={ localizeUrl( 'https://wordpress.com/support/quickstart-support/' ) }
+						>
+							Learn more
+						</Button>
+					</div>
+				</div>
+				<div className="help__support-session-illustration">
+					<img src={ supportSession } alt="" />
```

Comment:
We don't have fallbacks for these images?

Reply:
Usually we don't (My Home is an example) since they're not critical to understanding the content, but we require an empty alt attribute.